starter-code/iteration-2/main.ts

```diff
@@ -1,9 +1,40 @@
 // Add the reference to the "TodoInterface"
-
+interface Todo {
```

Comment:
Instead of puting the interface here write it in another file and import it in this one